packages/lib/server/getRoutedUrl.ts

```diff
@@ -32,6 +21,41 @@ function hasEmbedPath(pathWithQuery: string) {
   return onlyPath.endsWith("/embed") || onlyPath.endsWith("/embed/");
 }
 
+async function getRoutingFormModules() {
```

Comment:
All RAQB exports for some reason exports * or something that includes client code that we can't treeshake so this is the easiest way